cfgov/regulations3k/models/pages.py

```diff
@@ -461,7 +461,13 @@ def section_page(self, request, date_str=None, section_label=None):
             kwargs['date_str'] = date_str
 
         try:
-            section = section_query.get(label=section_label)
+            section = section_query.get(label__iexact=section_label)
+            if re.search('[A-Z]+', section_label):
+                return redirect(
+                    self.url + self.reverse_subpage(
+                        "index", kwargs=kwargs
+                    ) + section_label.lower() + '/'
+                )
```

Comment:
I'm pretty sure this should be a permanent redirect, but I'll check with our SEO experts to be sure.

```suggestion
            section = section_query.get(label__iexact=section_label)
            if re.search('[A-Z]+', section_label):
                return redirect(
                    self.url + self.reverse_subpage(
                        "index", kwargs=kwargs
                    ) + section_label.lower() + '/',
                    permanent=True
                )
```

Reply:
Fixed, I'm seeing a 301 redirect for uppercase-to-lowercase URLs in Chrome's dev tools.